### PR TITLE
Av 1128 consolidate gss

### PR DESCRIPTION
**Pull request by Aadityavoru**

### CODE REVIEW

File: champaign_county_map.html
Comment (mpkarpov-ui) [inline champaign_county_map.html:1]: This file looks like it's auto-generated.. any way we can `.gitignore` it and instead include steps to generate it?

File: ground/gss_combiner/README.md
Comment (mpkarpov-ui) [inline ground/gss_combiner/README.md:1]: This needs to be updated.. I will work on it once we settle on one final architecture for everything in this system.

File: ground/gss_combiner/test/test.py
Comment (mpkarpov-ui) [inline ground/gss_combiner/test/test.py:1]: TODO: Documentation for the null-modem emulator setup/operation.

File: ground/gss_combiner/util/mappy.py
Comment (mpkarpov-ui) [inline ground/gss_combiner/util/mappy.py:1]: Re: our conversation today-- we should move this out of the "gss combiner" space and more into "peripheral telemetry applications" space a la video system and live app.

File: offline_osm_map.html
Comment (mpkarpov-ui) [inline offline_osm_map.html:1]: This also looks autogenerated, see my comment on the first map.

**Review by mpkarpov-ui — COMMENTED** [review]
Looks pretty good so far but can use some cleanup changes.. we will also need to change feather code in accordance with new telemetry packets.

**Review by redindelible — CHANGES_REQUESTED** [review]
Don't commit files in __pycache__ (i.e. add **/*.pyc to .gitignore or something along those lines). 

Are the champaign _county_map.html or the bottomless map tiles required to be committed?

File: ground/gss_combiner/main.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        assert thread.is_alive()
+
+def threads_ready(threads: list[mqtt.TelemetryThread]):
+    """Used to ensure telemetry threads are ready"""
+    for thread in threads:
+        if not thread.ready():
+            return False
+    return True
+
+def parse_params(arguments):
```
Comment (mpkarpov-ui) [inline ground/gss_combiner/main.py:53]: let's replace this with argparse

File: ground/gss_combiner/util/mappy.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    print(lat, lon)
+    
+    if lat is not None and lon is not None:
+        with map_lock:
+            folium.Marker(location=[lat, lon]).add_to(mymap)
+
+
+def mqtt_thread():
+    global client 
+    broker_address = "10.195.167.19"
```
Comment (mpkarpov-ui) [inline ground/gss_combiner/util/mappy.py:27]: hard coded?

**Review by mpkarpov-ui — APPROVED** [review]
We should probably move some stuff from here to other locations (primarily map tiles). That being said as long as this works I'm ok having this being the software release for SG1.3

**Review by zyunlam — APPROVED** [review]

**Pull request merged**